lib/src/main/java/com/what3words/components/maps/wrappers/W3WGoogleMapsWrapper.kt

```diff
@@ -628,9 +651,9 @@ class W3WGoogleMapsWrapper(
     /** [getGridColorBasedOnZoomLevel] will get the grid color based on [GoogleMap.getCameraPosition] zoom. */
     private fun getGridSelectedBorderSizeBasedOnZoomLevel(zoom: Float = mapView.cameraPosition.zoom): Float {
         return when {
-            zoom < 18 -> context.resources.getDimension(R.dimen.grid_width_gone)
-            zoom >= 18 && zoom < 19 -> context.resources.getDimension(R.dimen.grid_selected_width_far)
-            zoom >= 19 && zoom < 20 -> context.resources.getDimension(R.dimen.grid_selected_width_middle)
+            zoom < 19 -> context.resources.getDimension(R.dimen.grid_width_gone)
```

Comment:
hey mate, we have the const DEFAULT_ZOOM_SWITCH_LEVEL, should we use it here instead of hardcoding the number?

Reply:
@phpduy yeah good point, not sure why it wasn't used before, will use it